1_fetch/src/fetch_data.R

```diff
@@ -0,0 +1,13 @@
+## Fetch data
+
+fetch_data <- function(project_output_dir, sb_item, sb_file, out_file) {
+  
+  #create output director, if it doesn't exist
+  if(!dir.exists(project_output_dir)){
+    dir.create(project_output_dir)
+  } 
+
+  #create file name  
+  mendota_file <- file.path(project_output_dir, out_file)
```

Comment:
Not necessary to change, just a note for the future. This function you wrote is very transferable to other applications, say if we had a different file from a different SB item that we wanted to download and save locally. That's great and a sign of truly modular code (perfect for pipelines and reusing later)! One thing that makes it less transferable (from a documentation/readability, not functionality perspective) is that you kept the object containing the local filepath as `mendota_file`. If we wanted to transfer this to a different file or SB item, that object name would likely not make sense anymore. I would suggest changing instances of `mendota_file` to `local_sb_file` or something slightly more generic.

Reply:
ah, got it. thanks!